README.md

```diff
@@ -247,6 +254,9 @@ nginx config, deproxy response, addr and port can use templates
 in format `${part_variable}` where `part` is one of 'server',
 'tempesta', 'client' or 'backend'
 
+Warning: deproxy backend now is running on the Framework host, so
+framework ip should be specified in tempesta config.
+
```

Comment:
The document doesn't explain the recent question with threading model where deproxy and a client are running in different threads.

Also it'd be very useful to write a high level section about most important classes. Imagine if you guide a new test developer where he should look in the framework to understand how a new test should be done.

Reply:
Yes, this is exactly my yesterday's request. Too lot of unexpected and hard to catch issues was found recently in test framework. We need a high level overview, where mock message is prepared, how it's sent to Tempesta, where and should be assertions, and so on. I bet, all that complicated issues can be noticed on that high level overview. Task planning should be also become simpler.

Reply:
write about this